2024-07-14 07:50:09,571 - INFO - Logging-System erfolgreich konfiguriert.
2024-07-14 07:50:29,414 - INFO - Logging-System erfolgreich konfiguriert.
2024-07-14 07:50:29,589 - ERROR - Error in main script: 
Traceback (most recent call last):
  File "/home/runneruser/actions-runner/_work/pdf-website/pdf-website/src/lib/test1.py", line 465, in <module>
    driver.get(user_link)
  File "/home/runneruser/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py", line 363, in get
    self.execute(Command.GET, {"url": url})
  File "/home/runneruser/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py", line 354, in execute
    self.error_handler.check_response(response)
  File "/home/runneruser/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_NAME_NOT_RESOLVED
  (Session info: chrome-headless-shell=126.0.6478.126)
Stacktrace:
#0 0x560dab8bfc7a <unknown>
#1 0x560dab5a2e2c <unknown>
#2 0x560dab59a9c0 <unknown>
#3 0x560dab58a4c2 <unknown>
#4 0x560dab58bb42 <unknown>
#5 0x560dab58a77d <unknown>
#6 0x560dab589e93 <unknown>
#7 0x560dab589d85 <unknown>
#8 0x560dab587cc6 <unknown>
#9 0x560dab5881ca <unknown>
#10 0x560dab5a5877 <unknown>
#11 0x560dab631f25 <unknown>
#12 0x560dab6125c2 <unknown>
#13 0x560dab631303 <unknown>
#14 0x560dab612363 <unknown>
#15 0x560dab5e2247 <unknown>
#16 0x560dab5e2b9e <unknown>
#17 0x560dab88624b <unknown>
#18 0x560dab88a2f1 <unknown>
#19 0x560dab871afe <unknown>
#20 0x560dab88ae52 <unknown>
#21 0x560dab85679f <unknown>
#22 0x560dab8af638 <unknown>
#23 0x560dab8af810 <unknown>
#24 0x560dab8bedac <unknown>
#25 0x7f1c4367cac3 <unknown>

2024-07-14 07:50:29,590 - INFO - 0.11622810363769531
2024-07-14 07:50:29,591 - INFO - Program finished. Total runtime: 0.11622810363769531 seconds.
2024-07-14 07:50:29,632 - ERROR - Fehler beim Planen des Cronjobs: CronTab.__init__() missing 1 required positional argument: 'crontab'
2024-07-14 07:50:29,634 - INFO - Program finished.
2024-07-14 07:50:29,635 - INFO - Program finished.
2024-07-14 07:56:10,663 - INFO - Logging-System erfolgreich konfiguriert.
2024-07-14 07:56:10,793 - ERROR - Error in main script: 
Traceback (most recent call last):
  File "/home/runneruser/actions-runner/_work/pdf-website/pdf-website/src/lib/test1.py", line 465, in <module>
    driver.get(user_link)
  File "/home/runneruser/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py", line 363, in get
    self.execute(Command.GET, {"url": url})
  File "/home/runneruser/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py", line 354, in execute
    self.error_handler.check_response(response)
  File "/home/runneruser/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_NAME_NOT_RESOLVED
  (Session info: chrome-headless-shell=126.0.6478.126)
Stacktrace:
#0 0x55e97a82dc7a <unknown>
#1 0x55e97a510e2c <unknown>
#2 0x55e97a5089c0 <unknown>
#3 0x55e97a4f84c2 <unknown>
#4 0x55e97a4f9b42 <unknown>
#5 0x55e97a4f877d <unknown>
#6 0x55e97a4f7e93 <unknown>
#7 0x55e97a4f7d85 <unknown>
#8 0x55e97a4f5cc6 <unknown>
#9 0x55e97a4f61ca <unknown>
#10 0x55e97a513877 <unknown>
#11 0x55e97a59ff25 <unknown>
#12 0x55e97a5805c2 <unknown>
#13 0x55e97a59f303 <unknown>
#14 0x55e97a580363 <unknown>
#15 0x55e97a550247 <unknown>
#16 0x55e97a550b9e <unknown>
#17 0x55e97a7f424b <unknown>
#18 0x55e97a7f82f1 <unknown>
#19 0x55e97a7dfafe <unknown>
#20 0x55e97a7f8e52 <unknown>
#21 0x55e97a7c479f <unknown>
#22 0x55e97a81d638 <unknown>
#23 0x55e97a81d810 <unknown>
#24 0x55e97a82cdac <unknown>
#25 0x7f1b683c4ac3 <unknown>

2024-07-14 07:56:10,794 - INFO - 0.09177017211914062
2024-07-14 07:56:10,794 - INFO - Program finished. Total runtime: 0.09177017211914062 seconds.
2024-07-14 07:56:10,817 - ERROR - Fehler beim Planen des Cronjobs: CronTab.__init__() missing 1 required positional argument: 'crontab'
2024-07-14 07:56:10,819 - INFO - Program finished.
2024-07-14 07:56:10,819 - INFO - Program finished.
2024-07-14 08:02:38,199 - INFO - Logging-System erfolgreich konfiguriert.
2024-07-14 08:03:04,878 - INFO - Logging-System erfolgreich konfiguriert.
2024-07-14 08:03:05,088 - ERROR - Error in main script: 
Traceback (most recent call last):
  File "/home/runneruser/actions-runner/_work/pdf-website/pdf-website/src/lib/test1.py", line 449, in <module>
    driver.get(user_link)
  File "/home/runneruser/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py", line 363, in get
    self.execute(Command.GET, {"url": url})
  File "/home/runneruser/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py", line 354, in execute
    self.error_handler.check_response(response)
  File "/home/runneruser/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_NAME_NOT_RESOLVED
  (Session info: chrome-headless-shell=126.0.6478.126)
Stacktrace:
#0 0x56092f8fdc7a <unknown>
#1 0x56092f5e0e2c <unknown>
#2 0x56092f5d89c0 <unknown>
#3 0x56092f5c84c2 <unknown>
#4 0x56092f5c9b42 <unknown>
#5 0x56092f5c877d <unknown>
#6 0x56092f5c7e93 <unknown>
#7 0x56092f5c7d85 <unknown>
#8 0x56092f5c5cc6 <unknown>
#9 0x56092f5c61ca <unknown>
#10 0x56092f5e3877 <unknown>
#11 0x56092f66ff25 <unknown>
#12 0x56092f6505c2 <unknown>
#13 0x56092f66f303 <unknown>
#14 0x56092f650363 <unknown>
#15 0x56092f620247 <unknown>
#16 0x56092f620b9e <unknown>
#17 0x56092f8c424b <unknown>
#18 0x56092f8c82f1 <unknown>
#19 0x56092f8afafe <unknown>
#20 0x56092f8c8e52 <unknown>
#21 0x56092f89479f <unknown>
#22 0x56092f8ed638 <unknown>
#23 0x56092f8ed810 <unknown>
#24 0x56092f8fcdac <unknown>
#25 0x7f6e72b61ac3 <unknown>

2024-07-14 08:03:05,090 - INFO - 0.15927791595458984
2024-07-14 08:03:05,090 - INFO - Program finished. Total runtime: 0.15927791595458984 seconds.
2024-07-14 08:03:05,117 - ERROR - Fehler beim Planen des Cronjobs: CronTab.__init__() missing 1 required positional argument: 'crontab'
2024-07-14 08:03:05,117 - INFO - Program finished.
2024-07-14 08:03:05,117 - INFO - Program finished.
2024-07-14 08:18:13,920 - INFO - Logging-System erfolgreich konfiguriert.
2024-07-14 08:19:03,763 - INFO - Logging-System erfolgreich konfiguriert.
2024-07-14 08:19:03,950 - ERROR - Error in main script: 
Traceback (most recent call last):
  File "/home/runneruser/actions-runner/_work/pdf-website/pdf-website/src/lib/test1.py", line 472, in <module>
    driver.get(user_link)
  File "/home/runneruser/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py", line 363, in get
    self.execute(Command.GET, {"url": url})
  File "/home/runneruser/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py", line 354, in execute
    self.error_handler.check_response(response)
  File "/home/runneruser/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_NAME_NOT_RESOLVED
  (Session info: chrome-headless-shell=126.0.6478.126)
Stacktrace:
#0 0x55cf62f89c7a <unknown>
#1 0x55cf62c6ce2c <unknown>
#2 0x55cf62c649c0 <unknown>
#3 0x55cf62c544c2 <unknown>
#4 0x55cf62c55b42 <unknown>
#5 0x55cf62c5477d <unknown>
#6 0x55cf62c53e93 <unknown>
#7 0x55cf62c53d85 <unknown>
#8 0x55cf62c51cc6 <unknown>
#9 0x55cf62c521ca <unknown>
#10 0x55cf62c6f877 <unknown>
#11 0x55cf62cfbf25 <unknown>
#12 0x55cf62cdc5c2 <unknown>
#13 0x55cf62cfb303 <unknown>
#14 0x55cf62cdc363 <unknown>
#15 0x55cf62cac247 <unknown>
#16 0x55cf62cacb9e <unknown>
#17 0x55cf62f5024b <unknown>
#18 0x55cf62f542f1 <unknown>
#19 0x55cf62f3bafe <unknown>
#20 0x55cf62f54e52 <unknown>
#21 0x55cf62f2079f <unknown>
#22 0x55cf62f79638 <unknown>
#23 0x55cf62f79810 <unknown>
#24 0x55cf62f88dac <unknown>
#25 0x7f4c2609bac3 <unknown>

2024-07-14 08:19:03,952 - INFO - 0.13863849639892578
2024-07-14 08:19:03,952 - INFO - Program finished. Total runtime: 0.13863849639892578 seconds.
